ang/civicase/contact/directives/contact-popover-content.directive.html

```diff
@@ -28,7 +28,20 @@ <h2>{{contact.display_name}}</h2>
   <div class="civicase__contact-popover__column">
     <div class="civicase__contact-popover__detail-group">
       <div class="civicase__contact-popover__detail-header"><strong>Primary Email</strong></div>
-      <div class="civicase__contact-popover__detail-value">{{contact.email}}</div>
+      <div class="civicase__contact-popover__detail-value">
+        <a
```

Comment:
Instead of duplicating the whole markup, lets send the parameters using a JS function.
`ng-href="{{ 'civicrm/activity/email/add' | civicaseCrmUrl:{ getParams() } }}"`